### PR TITLE
backport43 - GPG snippet related changes in client files

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Leftover change in the snippet has been identified.

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links
....

### CODE REVIEW

File: modules/client-configuration/pages/clients-almalinux.adoc [hunk 1]
```diff
@@ -148,14 +148,27 @@ For more information on activation keys, see xref:client-configuration:activatio
...
+
+== Manage GPG Keys
+
+Clients use GPG keys to check the authenticity of software packages before they are installed. 
+Only trusted software can be installed on clients.
+
+[IMPORTANT]
+====
+Trusting a GPG key is important for security on clients.
+It is the task of the adminstrator to decide which keys are needed and can be trusted.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-almalinux.adoc:167]:
```suggestion
It is the task of the administrator to decide which keys are needed and can be trusted.
```

File: modules/client-configuration/pages/clients-ubuntu-old.adoc [hunk 2]
```diff
@@ -5,8 +5,8 @@ This section contains information about registering Salt clients running {ubuntu
 
 ifeval::[{suma-content} == true]
 
-{productname} supports {ubuntu} 16.04 LTS, 18.04 LTS, and 20.04 LTS clients using Salt.
-For information about registering Salt clients running {ubuntu} 20.04, see xref:client-configuration:clients-ubuntu.adoc[].
+{productname} supports {ubuntu} 16.04 LTS, 18.04 LTS, 20.04 LTS and 22.04 LTS clients using Salt.
+For information about registering Salt clients running {ubuntu} 20.04 and 22.04, see xref:client-configuration:clients-ubuntu.adoc[].
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu-old.adoc:9]: seems to be unrelated to the GPG changes.

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
 [[clients-ubuntu]]
-= Registering {ubuntu} 20.04 Clients
+= Registering {ubuntu} 20.04 and 22.04 Clients
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc:2]: This and the following changes seem to be unrelated to the GPG changes.

File: modules/client-configuration/pages/snippets/trust_gpg.adoc [hunk 4]
```diff
@@ -73,7 +76,7 @@ You do not need to delete any previously stored keys.
 
 [IMPORTANT]
 ====
-Trusting a GPG key is important for security on clients.
-It is the task of the administrator to decide which keys are needed and can be trusted.
-A software channel cannot be assigned to a client when the GPG key is not trusted.
+Trusting a GPG key is important for security on clients. 
+It is the task of the admin to decide which keys are needed and can be trusted.
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```
Comment (keichwa) [inline modules/client-configuration/pages/snippets/trust_gpg.adoc:81]: I think the old version is better.

File: modules/client-configuration/pages/clients-almalinux.adoc [hunk 5]
```diff
@@ -148,14 +148,27 @@ For more information on activation keys, see xref:client-configuration:activatio
...
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-almalinux.adoc:168]: I proposed an accepted phrase in the old or existing `modules/client-configuration/pages/snippets/trust_gpg.adoc` file.  I'd appreciate if we could use this wording.  Also in `master` we probably should use this wording:
> It is the task of the administrator to decide which keys are needed and can be trusted.
> A software channel cannot be assigned to a client when the GPG key is not trusted.

**Review by keichwa — CHANGES_REQUESTED** [review]

**Review by jcayouette — APPROVED** [review]

**Pull request closed without merging**